lib/systems/default.nix

```diff
@@ -50,6 +50,7 @@ rec {
         else if final.isFreeBSD             then "fblibc"
         else if final.isNetBSD              then "nblibc"
         else if final.isAvr                 then "avrlibc"
+        else if final.isLoongArch64         then "loongarch"
```

Comment:
Sorry, but what was that for?

Reply:
It looks like a change when I was debugging... I checked, and `loongarch` seems using `glibc`, but I couldn't build it locally, just commit and look. (BC merge and other issue)